types/amap-js-api/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "only-arrow-functions": false,
```

Comment:
new definitions should just use the default lint rules

Reply:
Update now